src/activy/utils/stage/getStage.py

```
@@ -0,0 +1,80 @@
+import cv2
```

Ruff (0.8.2)

`cv2` imported but unused

Remove unused import: `cv2`

(F401)